Gems/ScriptCanvas/Code/Include/ScriptCanvas/Libraries/Math/MathExpression.h

```diff
@@ -41,7 +41,13 @@ namespace ScriptCanvas
                 ExpressionEvaluation::ParseOutcome ParseExpression(const AZStd::string& formatString) override;
 
                 AZStd::string GetExpressionSeparator() const override;
+
+            public:
+
+                static void ConfigureSlotDisplayType(ScriptCanvas::Node* node, ScriptCanvas::Slot* slot, AZ::Uuid type, AZStd::string name);
```

Comment:
Move up to the top of the file, with the other static functionality, in alphabetical order